Cinema/src/test/java/com/example/cinema/resource/ProjectionResourceTest.java

```diff
@@ -13,13 +18,18 @@
 import org.springframework.test.web.servlet.request.MockMvcRequestBuilders;
 import org.springframework.test.web.servlet.result.MockMvcResultMatchers;
 
+import java.time.LocalDate;
+import java.util.LinkedList;
+import java.util.List;
+
+import static org.mockito.Mockito.*;
```

Comment:
Avoid these. Always replace these with individual imports, to make sure you are only importing resources you need.

Reply:
Fixed.